libs/ui-watt/src/lib/foundations/color/colors.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * @Please notice!!! 
+ * These values should be kept in-sync with _colors.import.scss
+ * Naming convention is as follow: --watt-color-{COLOR GROUP (ONLY IF NOT THE SAME AS COLOR NAME)}-{COLOR NAME}
```

Comment:
Suggestion: Consider removing this line because it sounds confusing having a conditional in it.